component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/emailotp/EmailOTPAuthenticator.java

```diff
@@ -91,20 +95,156 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-import static org.wso2.carbon.identity.authenticator.emailotp.EmailOTPAuthenticatorConstants.DISABLE_OTP_RESEND_ON_FAILURE;
+import static org.wso2.carbon.identity.authenticator.emailotp.EmailOTPAuthenticatorConstants.ACCESS_TOKEN_REQUIRED_APIS;
+import static org.wso2.carbon.identity.authenticator.emailotp.EmailOTPAuthenticatorConstants.ACCOUNT_LOCKED_CLAIM;
```

Comment:
Will fix this issue according to the method suggested by Pamoda above.
The intention was to bring in all the constants as static imports. Since it seems to complicate the PR, I will revert it.
Thanks!